src/core/js/models/questionModel.js

```diff
@@ -1,343 +1,339 @@
         }
-      }
-    }
 
-    // Used to setup either global or local question weight/score
-    setupWeightSettings() {
-      // Not needed as handled by model defaults, keeping to maintain API
+        if (!componentButtons[key] && globalButtons[key]) {
+          componentButtons[key] = globalButtons[key];
+        }
+      }
     }
+  }
 
-    /// ///
-    // Submit process
-    /// /
+  // Used to setup either global or local question weight/score
+  setupWeightSettings() {
+    // Not needed as handled by model defaults, keeping to maintain API
+  }
 
-    // Use to check if the user is allowed to submit the question
-    // Maybe the user has to select an item?
-    canSubmit() {}
+  /// ///
+  // Submit process
+  /// /
 
-    checkCanSubmit() {
-      this.set('_canSubmit', this.canSubmit(), { pluginName: 'adapt' });
-    }
+  // Use to check if the user is allowed to submit the question
+  // Maybe the user has to select an item?
+  canSubmit() {}
 
-    // Used to update the amount of attempts the user has left
-    updateAttempts() {
-      if (!this.get('_attemptsLeft')) {
-        this.set('_attemptsLeft', this.get('_attempts'));
-      }
-      this.set('_attemptsLeft', this.get('_attemptsLeft') - 1);
-    }
+  checkCanSubmit() {
+    this.set('_canSubmit', this.canSubmit(), { pluginName: 'adapt' });
+  }
 
-    // Used to set _isEnabled and _isSubmitted on the model
-    setQuestionAsSubmitted() {
-      this.set({
-        _isEnabled: false,
-        _isSubmitted: true
-      });
+  // Used to update the amount of attempts the user has left
+  updateAttempts() {
+    if (!this.get('_attemptsLeft')) {
+      this.set('_attemptsLeft', this.get('_attempts'));
     }
+    this.set('_attemptsLeft', this.get('_attemptsLeft') - 1);
+  }
 
-    // Sets _isCorrect:true/false based upon isCorrect method below
-    markQuestion() {
+  // Used to set _isEnabled and _isSubmitted on the model
+  setQuestionAsSubmitted() {
+    this.set({
+      _isEnabled: false,
+      _isSubmitted: true
+    });
+  }
 
-      if (this.isCorrect()) {
-        this.set('_isCorrect', true);
-      } else {
-        this.set('_isCorrect', false);
-      }
+  // Sets _isCorrect:true/false based upon isCorrect method below
+  markQuestion() {
 
+    if (this.isCorrect()) {
+      this.set('_isCorrect', true);
+    } else {
+      this.set('_isCorrect', false);
     }
 
-    // Should return a boolean based upon whether to question is correct or not
-    isCorrect() {}
-
-    // Used to set the score based upon the _questionWeight
-    setScore() {}
+  }
 
-    // Checks if the question should be set to complete
-    // Calls setCompletionStatus and adds complete classes
-    checkQuestionCompletion() {
+  // Should return a boolean based upon whether to question is correct or not
+  isCorrect() {}
 
-      const isComplete = (this.get('_isCorrect') || this.get('_attemptsLeft') === 0);
+  // Used to set the score based upon the _questionWeight
+  setScore() {}
```

Comment:
add a space for curly brackets `isCorrect() { }` `setScore() { }`

Reply:
Same as above.